docs/DeveloperGuide.md

```diff
@@ -490,24 +417,25 @@ testers are expected to do more *exploratory* testing.
    1. Re-launch the app by double-clicking the jar file.<br>
        Expected: The most recent window size and location is retained.
 
-1. _{ more test cases …​ }_
+1. Shutting down app
+   1. Type in `exit` as the input or click the x on top left of app.
```

Comment:
If I'm not wrong, the X is on left for Mac but on right for Windows. Perhaps this part can be generalised by phrasing it as "clicking the X at the top of the app" or specifying left for Mac and right for windows?

Reply:
Oh didnt know bout this thnks!